src/autocapture.ts

```diff
@@ -37,6 +37,202 @@ function limitText(length: number, text: string): string {
     return text
 }
 
+export function getAugmentPropertiesFromElement(elem: Element): Properties {
```

Comment:
a bunch of functions move out of the class so we can use them directly because they (mostly) don't rely on state of the class

it should help with bundle size a tiny bit since the names are now minifiable